src/interfaces/jira.ts

```diff
@@ -0,0 +1,81 @@
+export interface JiraPullRequest {
+	commit: {
+		id: string;
+		repositoryUri: string;
+	};
+	ref: {
+		name: string;
+		uri: string;
+	}
+}
+
+export interface Build {
```

Comment:
Probably be good to prefix all interfaces with `Jira` to not cause any confusion.